game_coordinator/application/helpers/token_connect.py

```diff
@@ -162,6 +162,10 @@ async def _connect_direct_connect(self, server_ip, server_port):
         self._connect_method = f"direct-{ip_type}"
         self._connect_result_event.clear()
 
+        # Client requested an abort, but due to async behaviour we can still be exuected.
```

Comment:
```suggestion
        # Client requested an abort, but due to async behaviour we can still be executed.
```
and for all copy paste